int8 MODE_POSITION=0
int8 MODE_PRESET=1

int8 mode
float64 position
int32 preset
